extension/public/static/manifest/v3.json

```diff
@@ -3,17 +3,20 @@
   "version": "5.18.5",
   "version_name": "5.18.5",
   "description": "Freighter is a non-custodial wallet extension that enables you to sign Stellar transactions via your browser.",
+  "browser_specific_settings": {
```

Comment:
this is probably fine to leave in but looks like it's from a different line of work.